pkg/download/download.go

```diff
@@ -318,6 +321,18 @@ func createConfigsFromAPI(
 	return nil
 }
 
+func filterConfigsNotToDownload(a api.Api, values []api.Value) []api.Value {
+	result := make([]api.Value, 0, len(values))
+
+	for _, v := range values {
+		if !(a.GetId() == "anomaly-detection-metrics" && (strings.HasPrefix(v.Id, "dynatrace.") || strings.HasPrefix(v.Id, "ruxit."))) {
+			result = append(result, v)
+		}
```

Comment:
Could you please include some (debug) logging when a config is filtered out? 

And add a mention to the documentation that these are filtered and that any configs that where manually created with internal prefixes should be renamed - as we've seen when discussing it there's at least some chance that someone has created such a config via the API